resolwe_bio/tools/make_igv_session.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name,import-error
+"""Create IGV session."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import argparse
+from lxml import etree
+
+parser = argparse.ArgumentParser(description='Create igv session')
+
+parser.add_argument('-bam', '--bam_path', nargs='+', default=[], help='List of paths to bam file')
```

Comment:
If I understand the script correctly, bam files are required?

So I suggest to make them a positional argument (which is always required).
Another option would be to use the `required` keyword argument when calling `add_argument()`.

Are other arguments optional or also always required?

Another thing: I wouldn't be afraid to use really short option flags, e.g. `-g` for genome id and `-l` for locus.

Also, we typically follow the pattern to use double quotes for strings meant for human consumption and use single quotes for everything else. Since arguments and help messages are things that users of the tool will see, put them in double quotes.

To sum up, you could replace these 3 lines with the following:
```python
parser.add_argument("bam_files", nargs='+', metavar="BAM_FILE", help="List of paths to bam files")
parser.add_argument("-g", "--genomeid", help="Genome id")
parser.add_argument("-l", "--bam_locus", help="Locus where to open IGV")
```

if only bam files are required arguments which are treated as positonal arguments.

The tool's help would now look like:
```
usage: make_igv_session.py [-h] [-g GENOMEID] [-l BAM_LOCUS]
                           BAM_FILE [BAM_FILE ...]

Create igv session

positional arguments:
  BAM_FILE              List of paths to bam files

optional arguments:
  -h, --help            show this help message and exit
  -g GENOMEID, --genomeid GENOMEID
                        Genome id
  -l BAM_LOCUS, --bam_locus BAM_LOCUS
                        Locus where to open IGV
```